types/src/block_complete_entry.rs

```diff
@@ -1,7 +1,6 @@
 //! Contains [`BlockCompleteEntry`] and the related types.
 
 //---------------------------------------------------------------------------------------------------- Import
-#[cfg(feature = "epee")]
 use bytes::Bytes;
```

Comment:
`Bytes` is used in a field that _isn't_ gated by `epee`, so this needs to be removed.